pages/kubernetes/reference-content/introduction-to-kubernetes.mdx

```diff
@@ -119,33 +119,37 @@ A **service** is an abstraction which defines a logical group of pods that perfo
 
 By default, services are only available using internally routable IP addresses, but can be exposed publicly.
 
-It can be done either by using the `NodePort` configuration, which works by opening a static port on each node's external networking interface. Otherwise, it is possible to use the `LoadBalancer` service, which creates an external Load Balancer at a cloud provider using Kubernetes load-balancer integration.
+It can be done either by using the `NodePort` configuration, which works by opening a static port on each node's external networking interface. Otherwise, it is possible to use the `LoadBalancer` service, which creates an Scaleway Load Balancer using Kubernetes load-balancer integration, provided by the CCM.
```

Comment:
```suggestion
It can be done either by using the `NodePort` configuration, which works by opening a static port on each node's external networking interface. Otherwise, it is possible to use the `LoadBalancer` service, which creates a Scaleway Load Balancer using the Kubernetes load-balancer integration, provided by the CCM.
```